scrapers/5Kteens.yml

```
@@ -0,0 +1,31 @@
+name: 5Kteens
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - 5kteens.com/episodes
+      - 5kporn.com/episodes
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Studio:
+        Name:
+          fixed: 5Kteens
```

I don't know this site, but if you wanted to have the correct studio for each scene:
```suggestion
          selector: //span[starts-with(@class, "color-")]/@class
          postProcess:
            - replace:
                - regex: ^.+-
                  with:
            - map:
                5KT: 5Kteens
                5KP: 5Kporn
```